crates/jit/src/instantiate.rs

```diff
@@ -117,6 +123,25 @@ impl<'data> RawCompiledModule<'data> {
         // Make all code compiled thus far executable.
         compiler.publish_compiled_code();
 
+        // Create jitdump files
+        #[cfg(not(feature = "profiling"))]
+        let jit_dump_agent = None;
+        #[cfg(feature = "profiling")]
+        let jit_dump_agent = if _jitdump {
+            let agent = JitDumpAgent::new().ok();
+            let region_name = String::from("wasm_module");
```

Comment:
`translation.module.name` has access to optional module name.

Reply:
An outdated comment. Looks like this is not available after some refactoring but I agree we should look to replace this generic module name at some point in the future.